src/current/_data/redirects.yml

```
@@ -208,11 +208,15 @@
   versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1']
 
 - destination: demo-replication-and-rebalancing.md
-  sources: 
+  sources:
   - demo-automatic-rebalancing.md
   - demo-data-replication.md
   versions: ['v19.2', 'v20.1', 'v20.2', 'v21.1']
 
+- destination: disaster-recovery-planning.md
+  sources: ['disaster-recovery.md']
+  versions: ['v23.2']
```

```suggestion
- destination: v23.2/disaster-recovery-planning.md
  sources: ['v23.2/disaster-recovery.md']

- destination: dev/disaster-recovery-planning.md
  sources: ['dev/disaster-recovery.md']
```

There's a bug in the _redirects logic that's my fault. I'm gonna create a Jira ticket for this and work on it. For the time being, can you structure your redirect like this?